submissions/akulahere/DOM model/src/scripts/main.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const galleryList = document.querySelector('.gallery_list');
+const bigImage = document.querySelector('.gallery_large-img');
+const title = document.querySelector('figcaption');
+
+title.textContent = "Day and night"
+
+galleryList.addEventListener('click', (event) => {
+  event.preventDefault();
+  bigImage.src = event.target.closest('.list-item_link').href;
+  console.log(event.target.closest('.list-item_link').href);
```

Comment:
Is this for debugging purposes or part of the product?

Reply:
It was debugging info. I have removed it.